cmd/meroxa/turbine_cli/utils.go

```diff
@@ -447,3 +447,16 @@ func getTurbineJSBinary(params []string) []string {
 func executeTurbineJSCommand(params []string) *exec.Cmd {
 	return exec.Command(params[0], params[1:]...) //nolint:gosec
 }
+
+func shouldSkipDir(fi os.FileInfo) bool {
+	if !fi.IsDir() {
+		return false
+	}
+
+	switch fi.Name() {
+	case ".git", "fixtures", "node_modules":
```

Comment:
make sure we skip `node_modules` dir that a user may have locally installed for their app